ui/components/app/add-network/add-network.js

```diff
@@ -0,0 +1,134 @@
+import React, { useContext } from 'react';
+import PropTypes from 'prop-types';
+import { I18nContext } from '../../../contexts/i18n';
+import ActionableMessage from '../../ui/actionable-message';
+import Box from '../../ui/box';
+import Typography from '../../ui/typography';
+import {
+  ALIGN_ITEMS,
+  BLOCK_SIZES,
+  COLORS,
+  DISPLAY,
+  FLEX_DIRECTION,
+  TYPOGRAPHY,
+} from '../../../helpers/constants/design-system';
+import Button from '../../ui/button';
+
+const AddNetwork = ({
+  onBackClick,
+  onAddNetworkClick,
+  onAddNetworkManuallyClick,
+  featuredRPCS,
+}) => {
+  const t = useContext(I18nContext);
+
+  const nets = featuredRPCS
+    .sort((a, b) => (a.ticker > b.ticker ? 1 : -1))
+    .slice(0, 5);
+
+  return (
+    <Box>
+      <Box
+        height={BLOCK_SIZES.TWO_TWELFTHS}
+        padding={[4, 0, 4, 0]}
+        display={DISPLAY.FLEX}
+        alignItems={ALIGN_ITEMS.CENTER}
+        flexDirection={FLEX_DIRECTION.ROW}
+        className="add-network__header"
+      >
+        <img
+          src="./images/caret-left-black.svg"
+          alt={t('back')}
+          onClick={() => onBackClick}
```

Comment:
[nit]
```suggestion
          onClick={onBackClick}
```

Reply:
Does this even work as is (without invocation?)

Reply:
Must have done this in a hurry probably... Good catch! Thanks!
Fixed in: 7880226d2cbb1bdcffb7d9bbc1c996f7ec46dbac